lib/src/model/nutriments.dart

```diff
@@ -122,3 +194,71 @@ class Nutriments extends JsonObject {
 
   static Map<String, dynamic> toJsonHelper(Nutriments? n) => n?.toJson() ?? {};
 }
+
+class NutrimentValue {
```

Comment:
* Should be spelled `NutrientValue`
* But I'm not convinced of the added value of this class.